main.py

```diff
@@ -1,71 +1,45 @@
-import uuid
-
-from fastapi import Depends
-from fastapi import FastAPI, Form
-from fastapi import Request, Response
-from fastapi.responses import HTMLResponse
-from fastapi.templating import Jinja2Templates
-from sqlalchemy.orm import Session
+from sqlalchemy import Integer, String, Text
+from sqlalchemy.orm import Session, Mapped, mapped_column
 
 from database import Base
-from database import SessionLocal
-from database import engine
-from models import create_todo
-from models import delete_todo
-from models import get_todo
-from models import get_todos
-from models import update_todo
-
-Base.metadata.create_all(bind=engine)
 
-app = FastAPI()
-templates = Jinja2Templates(directory="templates")
 
+class ToDo(Base):
+    __tablename__ = "todos"
 
-def get_db():
-    db = SessionLocal()
-    try:
```

Comment:
The `update_todo` function now handles the `notes` field. Make sure to update any calling code to include this new parameter.